### PR TITLE
Added Discord and YouTube

### PR DESCRIPTION
**Pull request by anwesh-b**

### CODE REVIEW

**Comment by yankeexe** [comment]
@anwesh-b can you separate out the Actions on a different PR?

File: README.md [hunk 1]
```diff
@@ -80,5 +80,9 @@
 ## Discord Servers
 
 - [Docker](https://discord.gg/VhmRfsj)
+- [Kubernetes/Openshift](https://discord.gg/3F5nvYK)
+- [DevOps, SRE & Infrastructure](https://discord.com/invite/q49atz)
```
Comment (yankeexe) [inline README.md:84]: This link is invalid now.

File: README.md [hunk 2]
```diff
@@ -80,5 +80,9 @@
...
 
-## Youtube Channels
+## YouTube Channels
+
+- [TechWorld with Nana](https://www.youtube.com/channel/UCdngmbVKX1Tgre699)
```
Comment (yankeexe) [inline README.md:88]: This link returns a 404.

**Comment by anwesh-b** [comment]
Will create new PR seperating Workflow and Update The links

**Pull request closed without merging**